### PR TITLE
Add tests for blitting to cover width/height overflow.

### PR DESCRIPTION
**Pull request by Richard-Yunchao**
The range of an integer is [-0x7fffffff - 1, 0x7fffffff].
We restrict the width/height to the max value that can be
store in an integer in latest WebGL 2 spec. If we
reverse the data order for blitting, an invalid value like
0x7fffffff + 1 would become a valid value for an integer:
-0x7fffffff - 1. But it is invalid for blitting.

The new added tests cover this corner case to make it clear.

### CODE REVIEW

**Comment by jdashg** [comment]
I thought we forbade out-of-range blits?

**Comment by kainino0x** [comment]
For the record: this change follows on to https://github.com/KhronosGroup/WebGL/pull/2639

**Comment by jdashg** [comment]
I wish we would just forbid out-of-range blits, but ok.

File: sdk/tests/conformance2/rendering/blitframebuffer-size-overflow.html [hunk 1]
```diff
@@ -91,8 +91,12 @@
 
     gl.blitFramebuffer(-1, -1, max, max, 0, 0, width, height, gl.COLOR_BUFFER_BIT, gl.NEAREST);
     wtu.glErrorShouldBe(gl, gl.INVALID_VALUE, "Using source width/height greater than max 32-bit integer should fail.");
+    gl.blitFramebuffer(max, max, -1, -1, 0, 0, width, height, gl.COLOR_BUFFER_BIT, gl.NEAREST);
+    wtu.glErrorShouldBe(gl, gl.INVALID_VALUE, "Using source width/height greater than max 32-bit integer should fail.");
```
Comment (jdashg) [inline sdk/tests/conformance2/rendering/blitframebuffer-size-overflow.html]: I would prefer "greater than INT32_MAX".

**Comment by Richard-Yunchao** [comment]
Thanks for your review. Jeff and Kai.

I saw failures about these tests on Intel platforms after you changed the spec about blitting src/dst rect size at #2639 , so I revisited the tests and added a couple of tests here in order to make it clear.
